LanguageFeatures/Augmentation-libraries/augmenting_constructors_A01_t10.dart

```diff
@@ -37,26 +53,54 @@ enum E {
   const E.baz({required int x, final int y = 0}): x = x, y = y;
 }
 
+augment enum E {
+  ;
+  augment const E(x, [final y]);
+  augment const E.foo({required x, final y});
+  augment const E.bar(x, [final y]);
+  augment const E.baz({required x, final y});
+}
+
+
 extension type ET(int x) {
-  ET.foo(int this.x, [final int y = 0]);
-  ET.bar({required int this.x, final int y = 0});
-  ET.baz(int x, [final int y = 0]): x = x;
-  ET.qux({required int x, final int y = 0}): x = x;
+  ET.foo(int x, [final int y = 0]);
+  ET.bar({required int x, final int y = 0});
+  ET.baz(int x, [final int y = 0]);
+  ET.qux({required int x, final int y = 0});
+}
+
+augment extension type ET {
+  augment ET.foo(this.x, [final y]) {
+    x.expectStaticType<Exactly<int>>();
+    y.expectStaticType<Exactly<int>>();
+    _log = "Augmented: $x, $y";
```

Comment:
The `C` tests above don't write to the `_log` and there's no test case (old lines 50, 53, 56, 59). Is there a specific reason why it is helpful here but unnecessary in the `C` tests?

I'm guessing that it would be fine to add this element to the `C` tests as well.